### ISSUE TITLE
[IS 7.0][Docs] Text inside the info boxes are hard to read

### ISSUE THREAD

**Issue by pavinduLakshan** [wso2/product-is#19709]
The font size of the text inside info boxes in IS 7 docs are 12px, which makes it hard to read, especially for the visually impaired people.

![image](https://github.com/wso2/product-is/assets/41533942/a36dfc89-ce97-4064-95cb-0f624c5a7ee4)

The font size of the text outside the info boxes are 16px, which is better readable.

![image](https://github.com/wso2/product-is/assets/41533942/e677c79e-a32f-42d1-8925-46178290b446)

The general recommendation is to use at least 16px for body text.

https://accessibility.digital.gov/visual-design/typography/#:~:text=Use%20a%20large%20enough%20font,length%20that%20promotes%20comfortable%20reading.

**Comment by Achintha444** [comment]
Also I think the saturation of the text colour is bit much, which can cause the eye strain. 
To fix this we can try to reduce the saturation of the body text or increase the lightness of the background.

**Comment by SandaruRF** [comment]
Hi @himeshsiriwardana @sagara-gunathunga, I'd like to work on this issue. Could you please assign it to me?

**Comment by pavinduLakshan** [comment]
Hi @SandaruRF, really appreciate your enthusiasm on contributing to make WSO2 Identity Server product better. Let's move on to this task once you’ve wrapped up the other issues you’re currently working on.

**Comment by ashviniwe** [comment]
I'd like to work on this issue.

**Comment by Sanjaya-Samudra** [comment]
Yes, it is actually difficult to read. I will work on this issue as well.

**Comment by pavinduLakshan** [comment]
hi @ashviniwe, assigned the issue to you. Feel free to ping us if there's anything to clarify. Good luck!

**Comment by ashviniwe** [comment]
> hi [@ashviniwe](https://github.com/ashviniwe), assigned the issue to you. Feel free to ping us if there's anything to clarify. Good luck!

Thanks! I’ll reach out if I have any questions.

**Comment by pavinduLakshan** [comment]
Hi @ashviniwe, is there any update on the progress so far? Please don't hesitate to let us know in case if you are no longer able to work on this due to any reason. Thanks in advance!

**Comment by ashviniwe** [comment]
> Hi [@ashviniwe](https://github.com/ashviniwe), is there any update on the progress so far? Please don't hesitate to let us know in case if you are no longer able to work on this due to any reason. Thanks in advance!

Hi @pavinduLakshan, I have updated the PR. could you please check

**Comment by brionmario** [comment]
Hey @ashviniwe,

PR is now merged: https://github.com/wso2/docs-is/pull/5621

Thanks a lot for the contribution 🙌

**Comment by sagara-gunathunga** [comment]
@ashviniwe Congratulations on completing this issue! Unless you’ve already done so, please complete the form for this issue as mentioned on the WSO2 Hacktoberfest webpage: https://wso2.com/hacktoberfest/
